src/profiler/profiler.h

```diff
@@ -23,10 +23,14 @@
 #include "TSRM.h"
 #include "lib/lib.h"
 
+#define PROFILE_FILE_BUFFER_SIZE 131072
```

Comment:
safe to assume that nothing will be longer than `PROFILE_FILE_BUFFER_SIZE` / 2?

Reply:
It's never save to assume anything :-)